src/Microsoft.DotNet.Build.Tasks.net45/CreateExternalAssembliesResources.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections;
+using System.IO;
+using System.Reflection;
+using System.Resources;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class CreateExternalAssembliesResources : Task
+    {
+        [Required]
+        public ITaskItem[] InputAssemblies { get; set; }
+
+        [Required]
+        public string OutputPath { get; set; }
+
+        public override bool Execute()
+        {
+            try
+            {
+                if (!Directory.Exists(OutputPath))
+                {
+                    Directory.CreateDirectory(OutputPath);
+                }
+
+                CreateReswFiles();
+            }
+            catch (Exception e)
+            {
+                Log.LogErrorFromException(e, showStackTrace: false);
+            }
+
+            return !Log.HasLoggedErrors;
+        }
+
+        public void CreateReswFiles()
```

Comment:
I was under the impression that any PCL library that gets consumed in a UWP project would have its resources stripped out and put into a resw just the same way we are doing here which is why I figured there must already be a task to do it. If the task is too different or not available to our builds then putting this here is fine I just want to avoid duplication if we can.